### PR TITLE
Documented new menu.

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked to omichel/webots#5782

### CODE REVIEW

**Comment by DavidMansolino** [comment]
👍

File: guide/the-user-interface.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ The `Virtual Reality Headset` submenu allows you to use a virtual reality headse
 The `Enable` item allows you to switch the simulation view to the headset.
 The `Track headset position` and `Track headset orientation` items specifies if the headset position and orientation should be tracked and applied to the viewpoint.
 The `View left eye`, `View right eye` and `Empty view` radio buttons allow you to choose what should be displayed in the simulation view. You can either see the left eye image, right eye image or nothing.
+The `Anti-aliasing` item allows you to enable the anti-aliasing on both eyes images (note that anti-aliasing will decrease simulation speed).
```
Comment (DavidMansolino) [inline guide/the-user-interface.md]: Thank you, I was hesitating for the second (seems I picked the wrong one 😉 ).

Comment (tn12787) [inline guide/the-user-interface.md]: These things are a pain, for sure.

In general only one thing will take the plural form in these situations, and it's almost always the most concrete (and usually last) noun in the sentence.

**Comment by DavidMansolino** [comment]
👍

File: guide/the-user-interface.md (hunk 1 above)
Comment (DavidMansolino) [inline guide/the-user-interface.md]: Ok, thank you for the help, I will try to remember this rule.

**Pull request merged**